docker-compose.yml

```diff
@@ -13,6 +13,8 @@ services:
   mongo:
     image: mvertes/alpine-mongo
     restart: always
+    ports:
```

Comment:
Does this need to be included? I believe this will conflict with the existing mongo instance on the server that hosts automata

Reply:
For local development purposes yes, if you want I can remove it and document it elsewhere in the README.

---

Just curious, do you clone this project directly not have a separate docker-compose for production on your server?

Reply:
You're correct, I just clone the project directly and use the docker-compose file in it. All prod-specific config is in the .env file

Reply:
made changes to the docs to better describe the non-docker running of the bot, so removed this blob.